sdk/appconfiguration/Azure.Data.AppConfiguration/samples/README.md

```diff
@@ -0,0 +1,73 @@
+
+[Set Read Only Sample](Sample3_SetClearReadOnly.cs)
+
+## Read Revision History
+
+In this sample, we illustrate how to obtain the revision history of a configuration setting.  We first create a configuration setting, then make two separate updates to it.  Finally, we create a SettingSelector that uniquely identifies the configuration setting and use the selector to retrieve the configuration setting's revisions.  The sample uses a configuration setting with a timestamp in the key name to ensure the setting hasn't been used before and thereby minimize the size of the revision history.  It does not show that if you were to delete and recreate a configuration setting, the revision history would comprise both the history before the setting was deleted, and the history after it was recreated.
+
+[Read Revision History Sample](Sample4_ReadRevisionHistory.cs)
+
+## Get Setting If Changed
+
+In this sample, we show how to make the GetConfigurationSetting operation conditional, so that the request it sends to the App Configuration service is a [conditional request][conditional_request_mdn].  In the Azure SDK, [conditional requests are not sent unless opted-into][conditional_request_guideline].  Making the GetConfigurationSetting operation conditional means that the requested configuration setting will only be returned if it has changed in the configuration store since it was last retrieved, which can save bandwidth.  We show how to examine the operation response's status code to determine whether or not a configuration setting was returned in the body of the response.
+
+[Get Setting If Changed Sample](Sample5_GetSettingIfChanged.cs)
+
+## Update Setting If Unchanged
+
+In this sample, we show how to make the SetConfigurationSetting operation conditional, so that the request it sends to the App Configuration service is a [conditional request][conditional_request_mdn].  In the Azure SDK, [conditional requests are not sent unless opted-into][conditional_request_guideline].  Making the SetConfigurationSetting operation conditional allows us to ensure that the updates we made to the configuration setting are applied only if no other clients updated the configuration setting since the last time we retrieved it.  We walk through a hypothetical scenario where we release several virtual machines from our application's resource pool, and update the ``available_vms`` configuration setting to reflect this.  In this scenario, if another client were to have modified ``available_vms`` since we last retrieved it, and we updated it unconditionally, our update would overwrite their changes and the resulting value of ``available_vms`` would be incorrect.  We show how to implement optimistic concurrency to apply the update in a way that doesn't overwrite other clients' changes.
```

Comment:
Should only be one backtick to close and open, unless this actually is working (i.e. then nevermind).